src/core/annotation.js

```diff
@@ -704,6 +708,46 @@ class Annotation {
     }
   }
 
+  setRotation(mk) {
+    this.rotation = 0;
+    if (mk instanceof Dict) {
+      let angle = mk.get("R") || 0;
+      if (angle !== 0) {
```

Comment:
Let's ensure that the angle is also an *integer*, since otherwise none of the code below makes sense.
```suggestion
      if (Number.isInteger(angle) && angle !== 0) {
```